configs/jsx.js

```diff
@@ -4,6 +4,8 @@ export default [
   reactPlugin.configs.flat.recommended,
   {
     rules: {
+      'react/no-unknown-property': [ 'error', { ignore: [ 'class' ] } ],
+      'react/react-in-jsx-scope': 'off',
```

Comment:
Isn't that a rule we wanted to keep?

Reply:
We'd need to introduce it in a dedicated "react" profile, or enable it in a per project (i.e. Desktop Modeler only) basis.